datafiles/templates/Html/package-page.html.st

```diff
@@ -283,9 +260,37 @@
               <th>Status</th>
               <td>$buildStatus$</td>
             </tr>
+            <tr>
```

Comment:
I'm ok with moving to sidebar, but do not want it hidden by default. all the info except "edit package information" is generally relevant -- checking who the maintainers are, what the candidates are, etc.